server/bin/pbench-state-manager.py

```diff
@@ -19,7 +19,7 @@
 from pbench import BadConfig
 from pbench.common.logger import get_pbench_logger
 from pbench.server import PbenchServerConfig
-from pbench.server.database.database import Database
+from pbench.server.api import init_db
```

Comment:
`pbench.server.database`???

Reply:
@dbutenhof, what question are you asking or what sentiment are you expressing, here?

With these changes, I no longer see any reference to `pbench.server.database`, but I do see a (new) reference to `pbench.server.api:init_db`.

Reply:
> @dbutenhof, what question are you asking or what sentiment are you expressing, here?
> 
> With these changes, I no longer see any reference to `pbench.server.database`, but I do see a (new) reference to `pbench.server.api:init_db`.

Exactly; the other files have `from pbench.server.database import init_db` ... the new `init_db` is in `lib/pbench/server/database/__init__.py`, not in `lib/pbench/server/api/__init__.py`.

Reply:
yeah pushing the fix with other changes